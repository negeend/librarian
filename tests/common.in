COMMON 100000 100001
common 100000 100001
ADD MEMBER mem1
ADD MEMBER mem2
ADD MEMBER mem3
ADD MEMBER mem4
COMMON 100000 100001
coMmOn 100000 100001
ADD COLLECTION books.csv
ADD COLLECTION books2.csv
common 100000 100001
common 123455
COMMON 100000 100002 100000
RENT 100000 111111
RENT 100000 111112
RENT 100000 111113
RENT 100000 111114
RELINQUISH ALL 100000
RENT 100001 111112
RENT 100001 111113
RENT 100001 111114
COMMON 100000 100001
RELINQUISH ALL 100001
COMMON 100000 100001
rent 100002 111128
rent 100002 111129
rent 100002 111130
relinquish all 100002
MEMBER HISTORY 100000
MEMBER HISTORY 100001
MEMBER HISTORY 100002
Common 100000 100002
common 100001 100002
common 100000 100001
common 100000 100001 100002
rent 100003 111112
rent 100003 111113
RELINQUISH ALL 100003
common 100000 100001 100003
common 100000 100001 100002 100003
exit