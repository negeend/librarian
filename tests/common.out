user: No members in system.

user: No members in system.

user: Success.

user: Success.

user: Success.

user: Success.

user: No books in system.

user: No books in system.

user: 17 books successfully added.

user: 20 books successfully added.

user: No common books.

user: No such member in system.

user: Duplicate members provided.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: No common books.

user: Success.

user: The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)

user: The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: No common books.

user: No common books.

user: The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)
The Odyssey (Homer)

user: No common books.

user: Success.

user: Success.

user: Success.

user: The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: No common books.

user: Ending Library process.
